controllers/comments-controller.js

```diff
@@ -14,3 +15,20 @@ exports.getCommentsByArticle = (req, res, next) => {
             next(err);
         });
 };
+
+exports.postCommentByArticle = (req, res, next) => {
+    const {
+        body: { username, body },
+        params: { article_id },
+    } = req;
+    Promise.all([userByUsername(username), articleById(article_id)])
```

Comment:
article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown.